src/components/App.js

```diff
@@ -49,6 +50,23 @@ const App = () => (
       <Privateroute exact path="/article/new" component={Create} />
       <Privateroute exact path="/article/:slug/edit" component={Editarticle} />
       <Privateroute exact path="/bookmarks" component={Bookmarks} />
+      <Route exact path="/articles" component={MyArticles} />
+      <Route exact path="/article/new" component={Create} />
```

Comment:
These lines are duplicate

Reply:
Thank you for the feedback, but how are they duplicated?